### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by IvanKulundzic**
Initial setup of the project. Removed storyboard and unnecessary comments.

### CODE REVIEW

File: QuizApp/AppDelegate.swift [hunk 1]
```diff
@@ -9,28 +9,15 @@ import UIKit
...
-    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
-        // Called when the user discards a scene session.
-        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
-        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
-    }
-
-
+  
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    return true
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]: return is not needed

File: QuizApp/SceneDelegate.swift [hunk 2]
```diff
@@ -8,45 +8,32 @@
...
-        // Use this method to save data, release shared resources, and store enough scene-specific state information
-        // to restore the scene back to its current state.
-    }
-
-
+  
+  var window: UIWindow?
+  
+  func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    guard let scene = (scene as? UIWindowScene) else { return }
```
Comment (istaudacher) [inline QuizApp/SceneDelegate.swift]: is this indented properly?

**Review by istaudacher — CHANGES_REQUESTED** [review]

File: QuizApp/SceneDelegate.swift (hunk 2 above)
Comment (IvanKulundzic) [reply QuizApp/SceneDelegate.swift]: I've set the indentation and spacing to 2/2, as I prefer it, mostly for readability and compact code. Can it stay that way?
Screenshot 2022-07-12 at 13 36 28

Comment (istaudacher) [reply QuizApp/SceneDelegate.swift]: Please use 4 spaces as this is the team's standard.

Comment (IvanKulundzic) [reply QuizApp/SceneDelegate.swift]: Will revert the change.

File: QuizApp/SceneDelegate.swift [hunk 3]
```diff
@@ -11,42 +11,29 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
...
+    func sceneDidEnterBackground(_ scene: UIScene) {}
+}
 
-    func sceneDidEnterBackground(_ scene: UIScene) {
-        // Called as the scene transitions from the foreground to the background.
-        // Use this method to save data, release shared resources, and store enough scene-specific state information
-        // to restore the scene back to its current state.
+// MARK: - Private methods
+private extension SceneDelegate {
+    func setupInitialScene(with scene: UIWindowScene) {
```
Comment (istaudacher) [inline QuizApp/SceneDelegate.swift:34]: newline above & after functions

**Review by istaudacher — CHANGES_REQUESTED** [review]

File: QuizApp/SceneDelegate.swift (hunk 3 above)
Comment (IvanKulundzic) [reply QuizApp/SceneDelegate.swift:34]: Corrected throughout the project.

**Review by istaudacher — APPROVED** [review]

**Pull request merged**